scipy/io/mmio.py

```diff
@@ -30,7 +33,7 @@ def mminfo(source):
     Parameters
     ----------
 
-    source : file
+    filename : file
```

Comment:
My intention was to unify the parameter names of the `file-like` objects in the different methods. Sometimes it was `source` and sometimes `target`. I thought it would be more intuitive if its something like `file` or `file_like`.
Nevertheless it is an API change, so I am not sure what is the best.